diff --git a/linux/Makefile b/linux/Makefile
index 03f3af5..760e6ea 100644
--- a/linux/Makefile
+++ b/linux/Makefile
@@ -10,6 +10,9 @@ ccflags-y += -DHIGH_SPEED -Wno-unused-variable -Wno-declaration-after-statement
 all:
 	make -C $(KERNEL_DIR) M=$(ROOT_DIR) modules
 
+install:
+	make -C $(KERNEL_DIR) M=$(ROOT_DIR) modules_install
+
 clean:
 	-make -C $(KERNEL_DIR) M=$(ROOT_DIR) clean
 	-rm -f ../common/*.o ../common/.*.o.cmd
